taskfiles/deps/main.yaml

```diff
@@ -221,6 +222,21 @@ tasks:
           TARBALL_SHA256: "1250e4cc58bf06ee631567523f48848dc4596133e163f02615c97f78bab6c811"
           TARBALL_URL: "https://github.com/fmtlib/fmt/archive/refs/tags/10.2.1.tar.gz"
 
+  liblzma:
```

Comment:
nit: i think I prefer `lzma`, but it's really not important.

Reply:
I'm going by the library name in the package config file. LZMA is special in terms of its naming being prefixed with `lib`